third_party/conan/scripts/sync_dependencies.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+function conan_profile_exists {
+  conan profile show $profile >/dev/null 2>&1
+  return $?
+}
+
+
+REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../../" >/dev/null 2>&1 && pwd )"
+REPO_ROOT_WIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../../" >/dev/null 2>&1 && pwd -W 2>/dev/null)"
+
+# Path to script inside the docker container
+SCRIPT="/mnt/third_party/conan/scripts/sync_dependencies.sh"
+
+if [ "$1" ]; then
+  pip3 install conan==1.27.1
+  export QT_QPA_PLATFORM=offscreen
+
+  $REPO_ROOT/third_party/conan/configs/install.sh || exit $?
+  conan user -r artifactory $ARTIFACTORY_USERNAME -p $ARTIFACTORY_API_KEY || exit $?
+  if [ -n "$BINTRAY_USERNAME" -a -n "$BINTRAY_API_KEY" ]; then
+    conan remote enable bintray
+    conan user -r bintray $BINTRAY_USERNAME -p $BINTRAY_API_KEY || exit $?
+    conan remote disable bintray
```

Comment:
Yes, you're absolutely right. And it's even worse. The whole "if-bintray"-logic makes no sense since bintray-authentication is required for this use-case. All this was copied from the `build_and_upload_dependencies.sh` script. So I restructured all of this.